maps/atlas.dmm

```diff
@@ -2290,12 +2291,15 @@
 /turf/simulated/floor,
 /area/station/quartermaster/office)
 "gk" = (
-/obj/storage/secure/closet/engineering/cargo,
 /obj/cable{
 	d1 = 1;
 	d2 = 2;
 	icon_state = "1-2"
 	},
+/obj/machinery/computer/chem_requester/science{
+	dir = 8;
+	area_name = "Quarter Master's office"
```

Comment:
You don't necessarily need to assign this as it will pick up the area name automatically if left blank, but given how inconsistent different maps area names are this may make sense to do anyway 🤷

Reply:
Yeah I was getting some ambiguity that I wasn't a fan of, so I just standardised it.